diff --git a/sourceEnv.sh b/sourceEnv.sh
new file mode 100755
index 0000000..b17000a
--- /dev/null
+++ b/sourceEnv.sh
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+source /tools/Xilinx/Vivado/2024.1/settings64.sh
+source /home/user/petalinux/settings.sh
diff --git a/start_docker.sh b/start_docker.sh
new file mode 100755
index 0000000..81bdf07
--- /dev/null
+++ b/start_docker.sh
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+docker run \
+--init \
+--rm \
+-it \
+-v /tools/Xilinx:/tools/Xilinx:ro \
+-v /home/user/petalinux:/home/user/petalinux:ro \
+-v $(pwd):/AUP-ZU3 \
+--name pynq-sdbuild-env \
+--privileged \
+pynqdock:latest \
+/bin/bash
